src/components/scores/export.cjsx

```diff
@@ -134,6 +136,7 @@ ScoresExport = React.createClass
         isWaiting={ScoresExportStore.isExporting(courseId) or tryToDownload}
         isFailed={ScoresExportStore.isFailed(courseId) or downloadHasError}
         failedProps={failedProps}
+        timeout={3600000}
```

Comment:
The timeouts don't need to be an hour, they could probably be more like up to 5 minutes, because the jobs sometimes run longer than our default async button timeout